src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -144,7 +144,9 @@ public void RegexExceptionSerializable()
         {
             try
             {
+#pragma warning disable RE0001 // Regex issue: {0}
                 new Regex("*"); // parsing "*" - Quantifier {x,y} following nothing.
+#pragma warning restore RE0001 // Regex issue: {0}
```

Comment:
What is this?

Reply:
VS2019 has a new regex inspection feature which causes builds to fail with it ;)